definitions/npm/apollo-link-http_v1.2.x/flow_v0.28.x-/apollo-link-http_v1.2.x.js

```diff
@@ -0,0 +1,20 @@
+declare module "apollo-link-http" {
+  declare export interface FetchOptions {
+    uri?: string;
+    //todo change to type of Global['fetch']
+    fetch?: any;
+    includeExtensions?: boolean;
+    credentials?: string;
+    headers?: any;
+    fetchOptions?: any;
+  }
+
+  //todo change return type to ApolloLink from 'apollo-link'
+  declare export function createHttpLink(opts: FetchOptions): any;
```

Comment:
Despite the name, the `createHttpLink` function seems to return an `ApolloLink` which is from the `apollo-link` package. What is the appropriate way to declare this? (Keep in mind, `apollo-link` might not have lib defs either.)

Reply:
Same as in my last comment: [#1552 (discussion comment)](https://github.com/flowtype/flow-typed/pull/1552#discussion_r152002379)